docs/docs/guides/getting_started/introduction.md

```diff
@@ -22,6 +22,8 @@ The following documentation will guide you through different use cases of Web3.j
 -    Custom Output formatters
 -    In compliance with Eth EL API
 
+![Comparison of various package sizes](PackageSizeComparison.jpg)
```

Comment:
we should be showing (minified + gzipped) sizes @Deepak2030 Thanks

Reply:
Have updated, please check